helpers/index.tsx

```diff
@@ -30,6 +30,15 @@ export const getDate = () => {
   return date;
 };
 
+export const checkParams = (...params: any): boolean => {
```

Comment:
lets do some research about where to put helpers/utilities for APIs in nextjs.

Seems wrong to put both frontend and backend helpers in the same file.